tera-templates/core/home.html

```diff
@@ -0,0 +1,108 @@
+{%- extends "base.html" -%}
+
+{%- block title -%}Docs.rs{%- endblock title -%}
+
+{%- block body -%}
+    <div class="container landing">
+        <h1 class="brand"><i class="fa fa-cubes fa-fw"></i> Docs.rs</h1>
+
+        <form action="/releases/search" method="GET" class="landing-search-form">
+            <div>
+                <input class="search-input" id="search" name="query" type="text" aria-label="Find crate by search query"
+                    placeholder="Click or press 'S' to search" autofocus>
+            </div>
+
+            <div class="buttons">
+                <button type="submit" class="pure-button pure-button-normal">Search</button>
+                <button type="submit" class="pure-button pure-button-normal" id="i-am-feeling-lucky-button">
+                    I'm Feeling Lucky
+                </button>
+            </div>
+        </form>
+    </div>
+
+    <div class="container">
+        <div class="recent-releases-container">
+            <div class="release">
+                <a href="/releases">
+                    <strong>Recent Releases</strong>
+                </a>
+                <a href="/releases/feed" title="Atom feed">
+                    <i class="fa fa-rss-square"></i>
+                </a>
+            </div>
+
+            <ul>
+                {%- for release in recent_releases -%}
+                    {%- if rustdoc_status -%}
+                        {%- set release_url = "/" ~ release.name ~ "/" ~ release.version ~ "/" ~ release.target_name -%}
+                    {%- else -%}
+                        {%- set release_url = "/crate/" ~ release.name ~ "/" ~ release.version -%}
+                    {%- endif -%}
+
+                    <li>
+                        <a href="{{ release_url }}"
+                            class="release">
+                            <div class="pure-g">
+                                <div class="pure-u-1 pure-u-sm-6-24 pure-u-md-5-24 name">
+                                    {{ release.name }}-{{ release.version }}
+                                </div>
+                                <div class="pure-u-1 pure-u-sm-14-24 pure-u-md-16-24 description">
+                                    {{ release.description }}
+                                </div>
+
+                                {%- if varsb.show_stars -%}
+                                    <div class="pure-u-1 pure-u-sm-4-24 pure-u-md-3-24 date">
+                                        {{ release.stars }}
+                                        <i class="fa fa-star-o"></i>
+                                    </div>
+                                {%- else -%}
+                                    <div class="pure-u-1 pure-u-sm-4-24 pure-u-md-3-24 date">
+                                        {{ release.release_time | timeformat(relative=true) }}
```

Comment:
Can we stick a `title` with the absolute time on everywhere there is a relative time?